test/Sandpit/Exscientia/Trajectory/test_trajectory.py

```diff
@@ -0,0 +1,48 @@
+import BioSimSpace.Sandpit.Exscientia as BSS
+
+import pytest
+
+@pytest.fixture
+def traj_mdtraj(scope="session"):
+    """A trajectory object using the MDTraj backend."""
+    return BSS.Trajectory.Trajectory(
+            trajectory="test/Sandpit/Exscientia/input/trajectories/ala.trr",
+            topology="test/Sandpit/Exscientia/input/trajectories/ala.tpr")
```

Comment:
I think you mentioned that mdtraj cannot handle tpr as topology file?

Reply:
@lohedges Thanks. I wonder what do you mean `naming was irrelevant`. I would imagine that if you try to feed `tpr` with backend set to `mdtraj`, it should raise an error and say that it cannot read 'tpr' file?

Reply:
Yes, but now that we have the "auto" option it would just fallback to using `MDAnalysis`. In the tests, the naming was just to infer that I was passing files which would only work with a given backend, i.e. the named one, so I knew that I was testing two different backends, rather than the same one. I had effectively named them the round way around, which had no impact on the tests.